optuna/samplers/tpe/sampler.py

```diff
@@ -86,6 +86,38 @@ def objective(trial):
             study = optuna.create_study(sampler=TPESampler())
             study.optimize(objective, n_trials=10)
 
+    Args:
+        consider_prior:
+            Enhance the stability of Parzen estimator by imposing a Gaussian prior when True. The
+            prior is only effective if the sampling distribution is either `UniformDistribution`,
```

Comment:
Please use the `:class:...` notation to create a link to the distribution doc.
```suggestion
            prior is only effective if the sampling distribution is either :class:`~optuna.distributions.UniformDistribution`,
```